lib/redmine/scm/adapters/subversion_adapter.rb

```diff
@@ -125,11 +125,11 @@ def entries(path = nil, identifier = nil)
                                      kind: entry['kind'],
                                      size: ((s = entry['size']) ? s['__content__'].to_i : nil),
                                      lastrev: Revision.new(
-                              identifier: commit['revision'],
-                              time: Time.parse(commit_date['__content__'].to_s).localtime,
-                              author: ((a = commit['author']) ? a['__content__'] : nil)
-                              )
-                            )
+                                       identifier: commit['revision'],
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.